src/components/executive-education-2u/UserEnrollmentForm.jsx

```diff
@@ -0,0 +1,129 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  ActionRow, Button, Stack, Form, Hyperlink,
+} from '@edx/paragon';
+
+import { useActiveQueryParams } from './data';
+
+function UserEnrollmentForm({ className }) {
+  const [isTermsChecked, setIsTermsChecked] = useState(false);
+  const activeQueryParams = useActiveQueryParams();
+
+  return (
+    <form className={className}>
+      <div className="mb-4">
+        <h3 className="h4 mb-3">Personal information</h3>
+        <Stack direction="horizontal" gap={3}>
+          <Form.Group>
+            <Form.Control
+              value=""
+              onChange={() => {}}
+              floatingLabel="First Name"
+            />
+          </Form.Group>
+          <Form.Group>
+            <Form.Control
+              value=""
+              onChange={() => {}}
+              floatingLabel="Last Name"
+            />
+          </Form.Group>
+        </Stack>
+      </div>
+      <div className="mb-4">
+        <h3 className="h4 mb-3">Contact information</h3>
+        <Form.Group>
+          <Form.Control
+            value=""
+            onChange={() => {}}
+            floatingLabel="Address Line 1"
+          />
+        </Form.Group>
+        <Form.Group>
+          <Form.Control
+            value=""
+            onChange={() => {}}
+            floatingLabel="Address Line 2"
+          />
+        </Form.Group>
+        <Stack direction="horizontal" gap={3}>
+          <Form.Group>
+            <Form.Control
+              value=""
+              onChange={() => {}}
+              floatingLabel="City"
+            />
+          </Form.Group>
+          <Form.Group>
+            <Form.Control
+              value=""
+              onChange={() => {}}
+              floatingLabel="State"
+            />
+          </Form.Group>
+          <Form.Group>
+            <Form.Control
+              value=""
+              onChange={() => {}}
+              floatingLabel="Postal Code"
+            />
+          </Form.Group>
+        </Stack>
+        <Form.Group>
+          <Form.Control
+            value=""
+            onChange={() => {}}
+            floatingLabel="Country"
+          />
+        </Form.Group>
+      </div>
+      <div className="mb-4">
+        <Form.Group className="d-flex align-items-center">
+          <Form.Checkbox
+            checked={isTermsChecked}
+            onChange={() => setIsTermsChecked(!isTermsChecked)}
+          >
+            I agree
+          </Form.Checkbox>
+          &nbsp;to the&nbsp;
+          <Hyperlink
+            destination="https://www.getsmarter.com/terms-and-conditions-for-students"
+            target="_blank"
+          >
+            Terms and Conditions for Students
+          </Hyperlink>
```

Comment:
This is rendered outside of the `Form.Checkbox` label because its invalid to put interactive elements inside of labels for a11y. This is an open question for design (and Jeff W.) on how we want to handle this.